python/lsst/ts/FiberSpectrograph/avsFiberSpectrograph.py

```diff
@@ -220,7 +220,8 @@ def _connect(self, serial_number=None):
                 if serial_number == device.SerialNumber.decode("ascii"):
                     break
             else:
-                msg = f"Device serial number {serial_number} not found in device list: {device_list}"
+                msg = f"Device serial number {serial_number} not found in device list: {device_list}. "
+                "Check that the component deployed using the correct index?"
```

Comment:
Please change "deployed" to "is deployed" or something similar.

Also consider using `=` to shorten this, since we are finally on a version of Python that supports it and you are already changing this code:

```
msg = f"Device {serial_number=} not found in {device_list=}. "
```